src/james.data.clss.pas

```diff
@@ -207,13 +207,16 @@ function TDataStream.Save(const FileName: string): IDataStream;
 end;
 
 function TDataStream.AsString: string;
+var
+  Buf: AnsiString;
 begin
   with FStream do
   begin
-    SetLength(Result, Size);
+    SetLength(Buf, Size);
     Position := 0;
-    ReadBuffer(Pointer(Result)^, Size);
+    ReadBuffer(Pointer(Buf)^, Size);
```

Comment:
Actually, scratch that.
I had tested Delphi's way in FPC and it failed to compile, because FPC's implementation of TStringStream requires a constructor parameter, but if we put in an empty string as parameter, it will work in both FPC and Delph.
I'll make the modification and send you an updated PR.